<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Rest</class>
 <widget class="QWidget" name="Rest">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>741</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>0</y>
     <width>291</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Songti SC</family>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(0,0,0,0)</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:24pt; font-weight:700;&quot;&gt;沉浸式冥想 &lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QFrame" name="animationFrame">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>721</width>
     <height>551</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>200</width>
     <height>200</height>
    </size>
   </property>
   <property name="frameShape">
    <enum>QFrame::Shape::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Shadow::Raised</enum>
   </property>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>691</width>
      <height>541</height>
     </rect>
    </property>
    <property name="title">
     <string>音频选择</string>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>30</y>
       <width>91</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Songti SC</family>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgba(0,0,0,0)</string>
     </property>
     <property name="text">
      <string>音频选择</string>
     </property>
    </widget>
    <widget class="QComboBox" name="musicTypeComboBox_3">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>80</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">    QComboBox {
        background-color: white;               /* 主体白色背景 */
        border: 1px solid #b5d7b3;             /* 柔和绿色边框 */
        border-radius: 6px;
        padding: 4px 8px;
        font-size: 16px;
        color: #2e4e2e;                        /* 深绿色文字 */
    }

    QComboBox::drop-down {
        subcontrol-origin: padding;
        subcontrol-position: top right;
        width: 18px;
        border-left: 1px solid #b5d7b3;        /* 和外框协调 */
    }

    QComboBox::down-arrow {
	   image: url(:/icon/下拉箭头.jpeg);
        width: 12px;
        height: 12px;
    }

    QComboBox QAbstractItemView {
        background-color: white;               /* 下拉列表白色背景 */
        border: 1px solid #b5d7b3;
        selection-background-color: #d6f5d8;   /* 选中项浅绿色高亮 */
        selection-color: #2e4e2e;
        padding: 4px;
        font-size: 15px;
    }</string>
     </property>
     <item>
      <property name="text">
       <string>海浪</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>雨声</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>风铃</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>轻柔音乐</string>
      </property>
     </item>
    </widget>
    <widget class="QCheckBox" name="bedtimeModeCheckBox">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>130</y>
       <width>131</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Songti SC</family>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgba(0,0,0,0)</string>
     </property>
     <property name="text">
      <string>睡前模式</string>
     </property>
    </widget>
    <widget class="QPushButton" name="playButton">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>240</y>
       <width>81</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 2px solid #4CAF50; 
    border-radius: 10px;              /* 圆角边框，可改为0表示*/
    background-color: rgb(143, 255, 129);      
font: 700 18pt &quot;Kaiti SC&quot;;</string>
     </property>
     <property name="text">
      <string>播放</string>
     </property>
    </widget>
    <widget class="QPushButton" name="stopButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>240</y>
       <width>81</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 2px solid #4CAF50; 
    border-radius: 10px;              /* 圆角边框，可改为0表示*/
    background-color: rgb(143, 255, 129);      
font: 700 18pt &quot;Kaiti SC&quot;;</string>
     </property>
     <property name="text">
      <string>停止</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pauseButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>240</y>
       <width>81</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 2px solid #4CAF50; 
    border-radius: 10px;              /* 圆角边框，可改为0表示*/
    background-color: rgb(143, 255, 129);      
font: 700 18pt &quot;Kaiti SC&quot;;</string>
     </property>
     <property name="text">
      <string>暂停</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>280</y>
       <width>571</width>
       <height>101</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgba(0,0,0,0)</string>
     </property>
     <property name="title">
      <string>倒计时设置</string>
     </property>
     <widget class="QLabel" name="countdownLabel">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>81</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>倒计时: 5 分钟</string>
      </property>
     </widget>
     <widget class="QSlider" name="countdownSlider">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>541</width>
        <height>31</height>
       </rect>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>30</number>
      </property>
      <property name="value">
       <number>5</number>
      </property>
      <property name="orientation">
       <enum>Qt::Orientation::Horizontal</enum>
      </property>
     </widget>
     <widget class="QLabel" name="timeLabel">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>141</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>剩余时间: 05:00</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_7">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>400</y>
       <width>571</width>
       <height>111</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgba(0,0,0,0)</string>
     </property>
     <property name="title">
      <string>呼吸训练</string>
     </property>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>61</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>呼吸速度:</string>
      </property>
     </widget>
     <widget class="QSlider" name="breathSpeedSlider">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>521</width>
        <height>31</height>
       </rect>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>30</number>
      </property>
      <property name="pageStep">
       <number>1</number>
      </property>
      <property name="value">
       <number>2</number>
      </property>
      <property name="orientation">
       <enum>Qt::Orientation::Horizontal</enum>
      </property>
     </widget>
     <widget class="QLabel" name="breathSpeedLabel">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>80</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>速度：中等</string>
      </property>
     </widget>
    </widget>
    <widget class="QFrame" name="frame">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>30</y>
       <width>221</width>
       <height>181</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgba(0,0,0,0)</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::Shape::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Shadow::Raised</enum>
     </property>
     <widget class="QWidget" name="horizontalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>221</width>
        <height>181</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="layout"/>
     </widget>
    </widget>
    <zorder>label</zorder>
    <zorder>musicTypeComboBox_3</zorder>
    <zorder>bedtimeModeCheckBox</zorder>
    <zorder>playButton</zorder>
    <zorder>stopButton</zorder>
    <zorder>pauseButton</zorder>
    <zorder>groupBox_7</zorder>
    <zorder>groupBox_2</zorder>
    <zorder>frame</zorder>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
